docs/sources/helm-charts/mimir-distributed/_index.md

```diff
@@ -8,15 +8,18 @@ keywords:
   - Grafana Enterprise Metrics
   - Grafana metrics
 cascade:
-  # Used to generate relative URL paths to related documentation
   mimir_docs_version: "v2.9.x"
   gem_docs_version: "v2.9.x"
 ---
 
 # Grafana mimir-distributed Helm chart documentation
 
-The mimir-distributed Helm chart for [Grafana Mimir](/docs/mimir/{{< param "mimir_docs_version" >}}/) and [Grafana Enterprise Metrics](/docs/enterprise-metrics/{{< param "gem_docs_version" >}}/) (GEM) allows you to configure, install, and upgrade Grafana Mimir or Grafana Enterprise Metrics within a Kubernetes cluster.
+The mimir-distributed Helm chart for [Grafana Mimir] and [Grafana Enterprise Metrics](/docs/enterprise-metrics/{{< param "gem_docs_version" >}}/) (GEM) allows you to configure, install, and upgrade Grafana Mimir or Grafana Enterprise Metrics within a Kubernetes cluster.
```

Comment:
Why only Mimir here?

Reply:
Good question! I should have explained this in the PR description.

`docs/reference` does compile time link checking which means you have to have the documentation available which would be difficult for external contributors.

I think the future, it would be nice to update CI to do the full build and locally suppress the errors that relate to private content by default but expose an option for maintainers to do a full build since they can actually download the repository.

Reply:
Probably best for a future PR, this one is already complicated enough haha.

Reply:
Can you open a new issue stemming from your comment because it is confusing to only update one and not the other without context. I indeed like your idea about this.

Reply:
https://github.com/grafana/mimir/issues/5717 :tada: